search/search_repository.go

```diff
@@ -34,6 +34,10 @@ const (
 	IN       = "$IN"
 	SUBSTR   = "$SUBSTR"
 	WITGROUP = "$WITGROUP"
+	OPTS     = "$OPTS"
+
+	ParentExistsKey = "parent-exists"
+	TreeViewKey     = "tree-view"
```

Comment:
Please rename to `OptParentExists` and `OptTreeView`. Then we can get type completion in IDEs for just type `Opt<Ctrl>+<Space>`.

Reply:
Done. 04aa0e30c312e0f21ce1894185999e69f51beaa7